project/app/api/submission.py

```diff
@@ -1,91 +1,252 @@
 import logging
 from fastapi import APIRouter, File, UploadFile
+from fastapi.responses import JSONResponse
 from pydantic import BaseModel, Field, validator
-from app.utils.google_api import GoogleAPI, NoTextFoundException
+from app.utils.img_processing.google_api import GoogleAPI, NoTextFoundException
+from app.utils.complexity.squad_score import squad_score, scaler
+from hashlib import sha512
 import json
+from io import BytesIO
+from requests import get
+import sys
 
+# global variables and services
 router = APIRouter()
 log = logging.getLogger(__name__)
-# init the GoogleAPI service to fetch responses from the google vision api
 vision = GoogleAPI()
 
 
 class Submission(BaseModel):
-    """Data Model to parse Request body JSON, using the Default values
-    will check if the s3 API server is currently accepting requests"""
+    """
+    # Model that handles text submissions to the API `submission/text` endpoint
+    ## **Fields:**
+
+    ### SubmissionID - `int`
+    <p>
+        SubmissionID unique to this submission from a specific student
+    </p>
+
+    ### StoryId - `int`
```

Comment:
Are SubmissionID and StoryID the same? Is one redundant?

Reply:
Nope they are different, `StoryID` refers to the prompt number that changes once a week and `SubmissionID` refers to the student submission page for that prompt. I'll refactor the documentation her to clarify that.